ocsf-trimming/package.yaml

```diff
@@ -0,0 +1,46 @@
+id: ocsf-trimming
+name: OCSF Trimming
```

Comment:
Name of the package is debatable is aswell.

Reply:
Ideas:
- OCSF Size Management
- OCSF Volume Reduction
- OCSF Event Sizing

Reply:
Hmm, I'm not convinced. If you have a strong opinion about this please let me know, otherwise I'd just leave it at "OCSF Trimming".